PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-detail/pokemon-detail.component.scss

```diff
@@ -1,108 +1,17 @@
-.pokemon-detail-navigation {
```

Comment:
These styles should not be deleted, make sure the layout/design of the pokemon-detail navigation is not broken.